pytest/tests/mocknet/mirror.py

```diff
@@ -419,6 +421,14 @@ def start_traffic_cmd(args, traffic_generator, nodes):
     )
 
 
+def neard_runner_update_binaries(node):
+    neard_runner_jsonrpc(node, 'update_binaries')
+
+
+def update_binaries_cmd(args, traffic_generator, nodes):
+    pmap(neard_runner_update_binaries, nodes)
```

Comment:
should it also redownload on the traffic generator? with: `pmap(neard_runner_update_binaries, nodes + [traffic_generator])`